Game/Config/DefaultGame.ini

```diff
@@ -5,3 +5,5 @@ bSpatialNetworking=True
 
 [/Script/UnrealEd.ProjectPackagingSettings]
 +DirectoriesToAlwaysCook=(Path="Spatial")
+bCookAll=True
```

Comment:
Isn't this the default when not specifying maps? Can we just remove the `map` from the cook cmd line?

Reply:
Unfortunately it doesn't seem to be that way. When building by default (prior to this change) only the `EmptyGym` was packaged. I'm assuming it's controlled by either `Game Default Map` or `Server Default Map`: 
![image](https://user-images.githubusercontent.com/54983025/73278340-5bb21300-41e3-11ea-8f84-2f3f5a8e830e.png)

Reply:
Where's the call for cooking/packaging the game coming from?

Reply:
[This line](https://github.com/improbable/ShooterGameBuildKite/blob/9fe4275e2a10bc9a2a82e0d15fc2d62ecccb627c/ci/setup-and-build.ps1#L178) builds the server and [This line](https://github.com/improbable/ShooterGameBuildKite/blob/9fe4275e2a10bc9a2a82e0d15fc2d62ecccb627c/ci/setup-and-build.ps1#L194) in the same file builds the simplayer coordinator.

Reply:
Okay I follow